atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/TableMetadata.java

```diff
@@ -83,6 +83,15 @@ public LogSafety getNameLogSafety() {
         return LogSafety.UNSAFE;
     }
 
+    /**
+     * @return whether the table has densely accessed wide rows. This helps identify tables which could benefit from
+     * setting database tuning parameters to values more in line with such workflows, like the _transactions2 table.
+     */
+    @Value.Default
+    public boolean hasDenselyAccessedWideRows() {
+        return false;
```

Comment:
out of curiosity, if we were to expose this in the schema, is it safe to change after the fact? i.e. with a preexisting table?

Reply:
Yes, though the changes to the bloom filters / index summary etc. won't be applied to existing SSTables. The next time cassandra compacts those tables, it will write them out in the new format (so if you want to force it to happen now, as we did in benchmarks, you would run `nodetool compact`)

Reply:
interesting, nice to know!